flutter.json

```diff
@@ -0,0 +1,40 @@
+{
+    "homepage": "https://flutter.io/",
+    "version": "0.5.1",
+    "license": "https://github.com/flutter/flutter/blob/master/LICENSE",
+    "url": [
+        "https://storage.googleapis.com/flutter_infra/releases/beta/windows/flutter_windows_v0.5.1-beta.zip",
+        "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/flutter-dev-setup.ps1"
+    ],
+    "extract_dir": "flutter",
+    "hash": "1d6267358f90ad2ff46c23d53add0a503c10385efe6b5bb8c50ca4d670f9197a",
+    "depends": [
+        "android-sdk",
+        "https://raw.githubusercontent.com/se35710/scoop-java/master/oraclejdk8.json"
+    ],
+    "description": "Flutter is Google’s mobile app SDK for crafting high-quality native interfaces on iOS and Android in record time. This is the beta version, since Flutter wasn't officially 'released' yet.",
+    "env_add_path": "bin\\cache\\dart-sdk",
+    "bin": [
+        "bin\\flutter.bat",
+        "flutter-dev-setup.ps1"
```

Comment:
This is in `bin` & `post_install`? Why?

Reply:
In case it fails when post_installing. So the user can just run it again without looking for where it is. Do you think I should remove this?

Reply:
No, it’s just an interesting implementation.